packages/react-charts/src/components/ChartUtils/chart-theme.ts

```diff
@@ -113,8 +113,10 @@ export const getDonutThresholdStaticTheme = (
   invert?: boolean
 ): ChartThemeDefinition => {
   const staticTheme = cloneDeep(ChartDonutThresholdStaticTheme);
-  if (invert) {
-    staticTheme.pie.colorScale = staticTheme.pie.colorScale.reverse();
+  if (staticTheme.pie.colorScale instanceof Array) {
+    if (invert) {
```

Comment:
Would prefer to check `invert` first.

`if (invert && staticTheme.pie.colorScale instanceof Array) {`